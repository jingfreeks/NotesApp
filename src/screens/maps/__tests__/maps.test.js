import React from 'react';
import {render} from '@testing-library/react-native';
import MapsScreen from '../maps';


describe('Maps Screen', () => {
  it('Should work as expected to get snapshot', () => {
    const all = render(<MapsScreen />);
    expect(all.toJSON()).toMatchSnapshot();
  });
});
