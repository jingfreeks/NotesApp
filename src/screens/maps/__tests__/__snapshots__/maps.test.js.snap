// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Maps Screen Should work as expected to get snapshot 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "bottom": 0,
      "height": "100%",
      "justifyContent": "flex-end",
      "left": 0,
      "position": "absolute",
      "right": 0,
      "top": 0,
      "width": 400,
    }
  }
>
  <AIRMap
    initialRegion={null}
    onChange={[Function]}
    onMapReady={[Function]}
    region={null}
    style={
      {
        "bottom": 0,
        "left": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
      }
    }
  />
</View>
`;
